### PR TITLE
add /api/users get endpoint

### PR DESCRIPTION
**Pull request by andrewmar**

### CODE REVIEW

File: controllers/users.conteollers.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+const { selectAllUsers } = require("../models/users.models");
+
+exports.getAllUsers = (request, response, next) => {
+  selectAllUsers().then((users) => {
+    response.status(200).send({ users });
+  });
+};
```
Comment (two-triangles) [inline controllers/users.conteollers.js]: we should have a catch block just incase there is an error thrown

**Comment by two-triangles** [comment]
Looks good, just need to add that catch block and I am happy for you to merge!

File: __tests__/users.tests.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+          expect(typeof user.username).toBe("string");
+          expect(typeof user.name).toBe("string");
+          expect(typeof user.avatar_url).toBe("string");
+        });
+      });
+  });
+});
+
+describe("GET api/users", () => {
+  test("GET - status: 404 - when provided non existent endpoint", () => {
```
Comment (HannahBurks) [inline __tests__/users.tests.js:34]: This 404 is probably covered in your earlier tests for invalid end point, but you can leave this in if you want :)

**Review by HannahBurks — COMMENTED** [review]
You've already merged I see, but this is fine to move onto the next ticket :)

**Pull request merged**